wasmedge-mysql-nginx/compose.yml

```diff
@@ -0,0 +1,25 @@
+services:
+  frontend:
+    image: nginx:alpine
+    ports:
+      - 8090:80
+    volumes:
+      - ./frontend:/usr/share/nginx/html
+
+  backend:
+    image: demo-microservice
+    platform: wasi/wasm32
```

Comment:
```suggestion
    platform: wasi/wasm3
```

Reply:
I found that Docker Desktop 4.15 actually requires `wasi/wasm32`? Shall we wait until the next Docker Desktop release raise another PR to fix this? Thanks.

Reply:
@rumpl

Reply:
@juntao - How did you see that Docker Desktop requires `wasi/wasm32`? I've been using `wasi/wasm` in many of my PoCs and haven't run into issues. It's possible you need to rebuild the images after making the change, as you won't have a matching image with the right platform.

Reply:
My apologies. My Docker install had other problems before. I can confirm that `wasi/wasm` works on Docker Desktop 4.15. I have updated the code to reflect this. Thanks.

https://github.com/docker/awesome-compose/pull/309/commits/574e9807c6a12bf9bcd3b62deddd214ce4b11761